xcp_d/workflow/anatomical.py

```diff
@@ -229,1272 +222,886 @@ def init_anatomical_wf(
 
             from xcp_d.workflow.anatomical import init_anatomical_wf
             wf = init_anatomical_wf(
-                layout=None,
-                omp_nthreads=1,
                 fmri_dir=".",
                 subject_id="01",
                 output_dir=".",
-                input_type="fmriprep",
+                warp_to_standard=True,
+                omp_nthreads=1,
                 mem_gb=0.1,
                 name="anatomical_wf",
             )
 
     Parameters
     ----------
-    %(layout)s
     %(fmri_dir)s
     %(subject_id)s
     %(output_dir)s
-    %(input_type)s
+    warp_to_standard : :obj:`bool`
+        Whether to warp native-space surface files to standard space or not.
+        If False, the files are assumed to be in standard space already.
     %(omp_nthreads)s
     %(mem_gb)s
     %(name)s
         Default is "anatomical_wf".
 
     Inputs
     ------
-    t1w : str
-        Path to the T1w file.
-    t1seg : str
-        Path to the T1w segmentation file.
+    t1w_to_template_xform
+    template_to_t1w_xform
+    lh_inflated_surf, rh_inflated_surf
+        Left- and right-hemisphere inflated surface files.
+        Written out to derivatives if already in standard space, but unused otherwise.
+    lh_midthickness_surf, rh_midthickness_surf
+        Left- and right-hemisphere midthickness surface files.
+        Warped to standard space and written out to derivatives, but unused otherwise.
+    lh_pial_surf, rh_pial_surf
+        Left- and right-hemisphere pial surface files.
+        Warped to standard space, written out to derivatives,
+        and returned via outputnode for use in brainsprite.
+        Also used to generate HCP-style midthickness, inflated, and veryinflated surfaces
+        if they're not already available.
+    lh_smoothwm_surf, rh_smoothwm_surf
+        Left- and right-hemisphere smoothed white matter surface files.
+        Warped to standard space, written out to derivatives,
+        and returned via outputnode for use in brainsprite.
+        Also used to generate HCP-style midthickness, inflated, and veryinflated surfaces
+        if they're not already available.
 
     Notes
     -----
     If "hcp" or "dcan" input type, pre-generated surface files will be collected from the
     converted preprocessed derivatives.
     However, these derivatives do not include HCP-style surfaces.
+    Unless maybe they *are* HCP-style surfaces already?
 
-    If "fmriprep" or "nibabies", surface files in fsnative space will be extracted from the
-    associated Freesurfer directory (if available), and warped to fsLR space.
+    If "fmriprep" or "nibabies", surface files in fsnative space will be extracted from
+    the associated Freesurfer directory (if available), and warped to fsLR space.

```

Comment:
@madisoth the fMRIPrep anatomical derivatives include a T1w-to-fsnative transform. Just wondering, what is the purpose of that transform, and does it need to be incorporated in the anatomical workflow?

Reply:
It's the affine transform from subject T1w to FreeSurfer volume space ("fsnative" perhaps a misnomer as that's the surface space). The anatomical workflow doesn't need it (nor the inverse xfm)

Reply:
Thanks!